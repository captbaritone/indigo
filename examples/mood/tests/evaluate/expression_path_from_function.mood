fn func(): f64 {
    10_f64
}

pub fn test(): f64 {
    func::Bar
}