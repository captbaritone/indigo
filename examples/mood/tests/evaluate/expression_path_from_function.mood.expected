-----------------
INPUT
----------------- 
fn func(): f64 {
    10_f64
}

pub fn test(): f64 {
    func::Bar
}
-----------------
OUTPUT
-----------------
Error: Expected enum, got function:
 --> expression_path_from_function.mood:6:5

  | pub fn test(): f64 {
6 |     func::Bar
  |     ^^^^ Expected an enum.
  | }
